dot/build_spec_test.go

```diff
@@ -121,30 +124,27 @@ func TestBuildFromDB(t *testing.T) {
 	// initialise node (initialise state database and load genesis data)
 	cfg := NewTestConfig(t)
 	cfg.Init.Genesis = utils.GetGssmrGenesisRawPathTest(t)
-	nodeInstance := nodeBuilder{}
-	err := nodeInstance.initNode(cfg)
+	builder := nodeBuilder{}
+	err := builder.initNode(cfg)
 	require.NoError(t, err)
 
-	basePath := t.TempDir()
-	expectedPath := basePath[:len(basePath)-1] + "1"
+	path := t.TempDir()
+	basePath := path[:len(path)-1] + "1"
```

Comment:
by the way why do we do this? Why not use `t.TempDir()` directly? :thinking:

Reply:
This was done to replicate the basePath that is created by NewTestConfig, so this doesn't make sense since I can just read from cfg.Global.BasePath instead.  I've updated this.